actors/builtin/power/power_state.go

```diff
@@ -15,10 +15,12 @@ import (
 )
 
 type State struct {
-	TotalRawBytePower     abi.StoragePower
-	TotalQualityAdjPower  abi.StoragePower
-	TotalPledgeCollateral abi.TokenAmount
-	MinerCount            int64
+	TotalRawBytePower         abi.StoragePower
+	TotalRawBytePowerNoMin    abi.StoragePower
+	TotalQualityAdjPower      abi.StoragePower
+	TotalQualityAdjPowerNoMin abi.StoragePower
+	TotalPledgeCollateral     abi.TokenAmount
+	MinerCount                int64
```

Comment:
While you're here, move `NumMinersMeetingMinPower` next to `MinerCount`. We've copped some flak for renaming things, but I'd suggest `MinerAboveMinPowerCount` or similar.